### ISSUE TITLE
Does the SD3 branch support fine-tuning CLIP L and G?

### ISSUE THREAD

**Issue by Ice-YY** [kohya-ss/sd-scripts#1840]
Hi,
I was trying to finetune SD3.5M with CLIP L and G. I used full BF16 with  fused backward pass option (block swap and cache text encoders were both disabled) and the AdaFactor optimizer with a batch size of 16. The learning rate for the text encoder was set to 2e-6 (scheduler constant with warmup). However, when I checked the output model after 8,000 steps, the CLIP models were completely broken. Do you think the learning rate might be too high for the text encoder, or is CLIP finetuning not fully supported?
Thanks!
